istio/tests/test_istio_v1.py

```diff
@@ -84,3 +84,16 @@ def test_legacy_version_metadata(datadog_agent, dd_run_check):
     }
 
     datadog_agent.assert_metadata('test:123', version_metadata)
+
+
+def test_legacy_istio_agent(aggregator, dd_run_check, mock_http_response):
```

Comment:
Looks like this test is failing for py27. I believe it also needs to be skipped for py27

Reply:
I fixed it, used the wrong instance config.